source/coverage.cmake

```diff
@@ -0,0 +1,3 @@
+if(YOTTA_CFG_DEBUG_OPTIONS_COVERAGE_MODULES_SOCKETS)
+  set( CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -fprofile-arcs -ftest-coverage" )
```

Comment:
Potentially, or is that documented behaviour of cmake? The reason I didn't use target properties is that it requires knowing what the target is.